### PR TITLE
feat(sbb-pearl-chain-time): add extended transfer render logic

### PR DESCRIPTION
**Pull request by osminaz**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

-

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/components/sbb-pearl-chain/sbb-pearl-chain.helper.tsx [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import { PtRideLeg } from '../../global/interfaces/pearl-chain-properties';
-
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export const isRideLeg = (leg: any): leg is Extract<PtRideLeg, { __typename: 'PTRideLeg' }> => {
+export const isRideLeg = (leg: any): leg is Extract<any, { __typename: 'PTRideLeg' }> => {
```
Comment (jeripeierSBB) [inline src/components/sbb-pearl-chain/sbb-pearl-chain.helper.tsx:2]: Can `any` really not be prevented?

File: src/components/sbb-pearl-chain/sbb-pearl-chain.sample-data.js [hunk 2]
```diff
@@ -75,6 +75,27 @@ export const futureLeg = {
...
+  departure: {
+    time: future,
+  },
+  serviceJourney: {
+    ...defaultService,
+    notices: [
+      {
+        name: 'CI',
+        text: {
+          template: "Verlängerte Einsteigezeit (45')",
```
Comment (jeripeierSBB) [inline src/components/sbb-pearl-chain/sbb-pearl-chain.sample-data.js:92]: Maybe english?

File: src/components/sbb-pearl-chain-time/sbb-pearl-chain-time.stories.js [hunk 3]
```diff
@@ -84,12 +84,22 @@ export const maximal = Template.bind({});
 maximal.argTypes = defaultArgTypes;
 maximal.args = {
```
Comment (jeripeierSBB) [inline src/components/sbb-pearl-chain-time/sbb-pearl-chain-time.stories.js:85]: Is maximal still the correct title as there is not the "maximal" content :P

File: src/components/sbb-pearl-chain/sbb-pearl-chain.helper.tsx (hunk 1 above)
Comment (osminaz) [reply src/components/sbb-pearl-chain/sbb-pearl-chain.helper.tsx:2]: I removed the hole helper because we need it only one

**Pull request merged**